src/proposals/proposalTypes/HybridProposal.sol

```diff
@@ -551,8 +624,105 @@ abstract contract HybridProposal is
         );
 
         {
+            address wormholeCoreBase = addresses.getAddress(
+                "WORMHOLE_CORE_BASE",
+                baseChainId
+            );
+            address wormholeCoreBaseSepolia = addresses.getAddress(
+                "WORMHOLE_CORE_SEPOLIA_BASE",
+                baseSepoliaChainId
+            );
+            address temporalGov = addresses.getAddress(
+                "TEMPORAL_GOVERNOR",
+                sendingChainIdToReceivingChainId[block.chainid]
+            );
+            address wormholeCoreMoonbeam = block.chainid == moonBeamChainId
+                ? addresses.getAddress(
+                    "WORMHOLE_CORE_MOONBEAM",
+                    moonBeamChainId
+                )
+                : addresses.getAddress(
+                    "WORMHOLE_CORE_MOONBASE",
+                    moonBaseChainId
+                );
+
```

Comment:
Could we get both addresses for Moonbeam/Moonbase and check them both, or alternatively, change the logic for wormholeCoreBase to make it consistent with this approach?